### ISSUE TITLE
There is one suggestion.

### ISSUE THREAD

**Issue by kjh2671** [featurecat/lizzie#308]
How about add a function to record the winning rate and visit count to sgf?
I think it will help people review again later.

**Comment by featurecat** [comment]
yes... this is a top requested feature. I am not working on it at the moment, but it is a top priority. lately I don't have much free time to work on lizzie. it depends on the contributions of others or when I have time to do it myself.
